client/js/components/segment/SegmentImportJobList.vue

```diff
@@ -0,0 +1,317 @@
+<template>
```

Comment:
I think this file should be moved to `client/procedure/AdministrationImport`, and the `segment` directory should be removed

Reply:
indeed, thx 277c423b